fe/fe-core/src/main/java/org/apache/doris/analysis/MVColumnItem.java

```diff
@@ -108,6 +108,7 @@ public Column toMVColumn(OlapTable olapTable) throws DdlException {
             Preconditions.checkNotNull(defineExpr != null);
             Column result = new Column(name, type, isKey, aggregationType, ColumnDef.DefaultValue.ZERO, "");
             result.setDefineExpr(defineExpr);
+            result.setUniqueId(olapTable.incAndGetMaxColUniqueId());
```

Comment:
In the future, this change will cause problem when we want to support mv do light schema change and olapTable.incAndGetMaxColUniqueId() need to write editlog for duration.   
the commit  can justly use temporaryly.

Reply:
could you try like this in   MaterializedViewHandler.java  checkAndPrepareMaterializedView()

        for (Column column : rollupSchema) {
            column.setUniqueId(Column.COLUMN_UNIQUE_ID_INIT_VALUE);
        }

Reply:
> could you try like this in MaterializedViewHandler.java checkAndPrepareMaterializedView()
> 
> ```
>     for (Column column : rollupSchema) {
>         column.setUniqueId(Column.COLUMN_UNIQUE_ID_INIT_VALUE);
>     }
> ```

Hi, Do you can review #10792? I set uniqid also.

Reply:
@ the change will override yours

> > could you try like this in MaterializedViewHandler.java checkAndPrepareMaterializedView()
> > ```
> >     for (Column column : rollupSchema) {
> >         column.setUniqueId(Column.COLUMN_UNIQUE_ID_INIT_VALUE);
> >     }
> > ```
> 
> Hi, Do you can review #10792? I set uniqid also.

 the change will override yours